Patches/More Vanilla Turrets/MVT_CE_Patch_Buildings_Security.xml

```diff
@@ -0,0 +1,755 @@
+				<value>
+					<turretBurstCooldownTime>15</turretBurstCooldownTime>
+				</value>
+			</li>
+
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Bullet_81mmMortarShell_Plasma"]/graphicData/texPath</xpath>
+				<value>
+					<texPath>Things/Building/Security/DevastatorShell</texPath>
+				</value>
+			</li>
+
+			<!-- ========== Common to many or all Turrets ========== -->
+
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[
+					defName="GunComplex" or
+					defName="RocketComplex" or
+					defName="MilitaryTurretGun" or
+					defName="MilitaryTurretGunManned" or
+					defName="ShredderTurretGun" or
+					defName="ShredderTurretGunManned" or
+					defName="PrecisionTurretGun" or
+					defName="PrecisionTurretGunManned" or
+					defName="BlastTurretGun" or
+					defName="BlastTurretGunManned" or
+					defName="VulcanCannon" or
+					defName="VulcanCannonManned" or
+					defName="Turret_DevastatorMortarBomb"
+				]/thingClass</xpath>
+				<value>
+					<thingClass>CombatExtended.Building_TurretGunCE</thingClass>
+				</value>
+			</li>
+
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[
+					defName="MilitaryTurretGun" or
+					defName="MilitaryTurretGunManned" or
+					defName="ShredderTurretGun" or
+					defName="ShredderTurretGunManned" or
+					defName="PrecisionTurretGun" or
+					defName="PrecisionTurretGunManned" or
+					defName="BlastTurretGun" or
+					defName="BlastTurretGunManned"
+				]/fillPercent</xpath>
+				<value>
+					<fillPercent>0.85</fillPercent>
+				</value>
+			</li>
+
+			<li Class="PatchOperationRemove">
+				<xpath>Defs/ThingDef[
+					defName="GunComplex" or
+					defName="RocketComplex" or
+					defName="MilitaryTurretGun" or
+					defName="MilitaryTurretGunManned" or
+					defName="ShredderTurretGun" or
+					defName="ShredderTurretGunManned" or
+					defName="PrecisionTurretGun" or
+					defName="PrecisionTurretGunManned" or
+					defName="BlastTurretGun" or
+					defName="BlastTurretGunManned" or
+					defName="VulcanCannon" or
+					defName="VulcanCannonManned"
+				]/comps/li[@Class = "CompProperties_Refuelable"]</xpath>
+			</li>
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[
+					defName="GunComplex" or
+					defName="RocketComplex" or
+					defName="MilitaryTurretGun" or
+					defName="MilitaryTurretGunManned" or
+					defName="ShredderTurretGun" or
+					defName="ShredderTurretGunManned" or
+					defName="PrecisionTurretGun" or
+					defName="PrecisionTurretGunManned" or
+					defName="BlastTurretGun" or
+					defName="BlastTurretGunManned" or
+					defName="VulcanCannon" or
+					defName="VulcanCannonManned"
+				]/statBases</xpath>
+				<value>
+					<AimingAccuracy>1</AimingAccuracy>
+					<ShootingAccuracyTurret>1</ShootingAccuracyTurret>
+				</value>
+			</li>
+
+			<li Class="PatchOperationRemove">
+				<xpath>Defs/ResearchProjectDef[defName="VulcanCannon"]/prerequisites/li[text()="MultibarrelWeapons"]</xpath>
```

Comment:
It should be alright with that change, since the patch will leave the `BlastTurret` prerequisite untouched:

```
  <ResearchProjectDef>
    <defName>VulcanCannon</defName>
    <label>vulcan cannon</label>
    <description>Allows the production of Vulcan cannons.</description>
    <baseCost>6000</baseCost>
    <techLevel>Industrial</techLevel>
    <prerequisites>
      <li>BlastTurret</li>
      <li>MultibarrelWeapons</li>  <!-- Removed by CE patch-->
    </prerequisites>
    <requiredResearchBuilding>HiTechResearchBench</requiredResearchBuilding>
    <requiredResearchFacilities>
      <li>MultiAnalyzer</li>
    </requiredResearchFacilities>
    <tab>MVT</tab>
    <researchViewX>5</researchViewX>
    <researchViewY>1</researchViewY>
  </ResearchProjectDef>
```